pkg/installer/install.go

```diff
@@ -37,12 +39,15 @@ func (m *manager) Install(ctx context.Context) error {
 		steps.Action(func(ctx context.Context) error {
 			var err error
 			// Applies ARO-specific customisations to the InstallConfig
-			g, err = m.applyInstallConfigCustomisations(ctx, installConfig, image)
+			g, err = m.applyInstallConfigCustomisations(installConfig, image)
 			return err
 		}),
 		steps.Action(func(ctx context.Context) error {
 			return m.persistGraph(ctx, g)
 		}),
+		steps.Action(func(ctx context.Context) error {
+			return m.determineOutboundType(m.oc, m.sub)
+		}),
```

Comment:
This has to happen before `m.applyInstallConfigCustomisations` otherwise the field will be nil.